src/WebUI/Startup.cs

```diff
@@ -45,6 +45,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllersWithViews(options =>
                 options.Filters.Add(new ApiExceptionFilter()));
 
+            // Adds timezone handling in requests for date strings, src: https://stackoverflow.com/a/60785033 
+            services.AddControllers().AddNewtonsoftJson(
+                options => options.SerializerSettings.DateTimeZoneHandling = Newtonsoft.Json.DateTimeZoneHandling.Utc);
```

Comment:
```suggestion
                options.Filters.Add(new ApiExceptionFilter()))
                // Adds timezone handling in requests for date strings, src: https://stackoverflow.com/a/60785033 
                .AddNewtonsoftJson(
                    options => options.SerializerSettings.DateTimeZoneHandling = Newtonsoft.Json.DateTimeZoneHandling.Utc);
```

turns out the second "AddControllers" isn't actually necessary